modules/openshift-developer-cli-deleting-an-application.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.adoc
+// * cli_reference/openshift_developer_cli/creating-an-application-with-a-database.adoc
+// * cli_reference/openshift_developer_cli/creating-a-multicomponent-application-with-odo.adoc
+
+[id="deleting-an-application_{context}"]
+
+= Deleting an Application
+
+[IMPORTANT]
+====
+Deleting the application will delete all components associated with the application.
+====
+
+.Procedure
+
+. List the applications in the current project:
++
+----
+$ odo app list
+----
+
+. List the components associated with the applications:
++
+----
+$ odo component list
+    APP     NAME                      TYPE       SOURCE        STATE
+    app     nodejs-nodejs-ex-elyf     nodejs     file://./     Pushed
+----
+
+. Delete the application, confirm the deletion with `Y`:
++
+[NOTE]
+====
+Suppress the confirmation prompt with `-f` flag.
+====
++
+----
+$ odo app delete <application_name>
+    ? Are you sure you want to delete the application: <application_name> from project: <project name>
+----
+
+. The app is deleted.
```

Comment:
This is not a step. It is also unnecessary to mention it, suggest removing it.

Reply:
Yes, I was not sure about this one.

Reply:
Please remove it.

Reply:
done!